docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -0,0 +1,255 @@
+the machine config operator rebooting each node sequentially, according to your machine config pool configuration:
+```console
+$ oc patch featuregates/cluster --type=merge --patch='{"spec":{"featureSet":"CustomNoUpgrade","customNoUpgrade":{"enabled":["GatewayAPI"]}}}'
+featuregate.config.openshift.io/cluster patched
+```
+
+Wait for the ingress operator to successfully install the Gateway API CRDs:
+```console
+$ oc get crd gatewayclasses.gateway.networking.k8s.io httproutes.gateway.networking.k8s.io gateways.gateway.networking.k8s.io referencegrants.gateway.networking.k8s.io
+NAME                                       CREATED AT
+gatewayclasses.gateway.networking.k8s.io   2023-04-04T18:02:55
+httproutes.gateway.networking.k8s.io        2023-04-04T18:02:55Z
+gateways.gateway.networking.k8s.io          2023-04-04T18:02:55Z
+referencegrants.gateway.networking.k8s.io   2023-04-04T18:02:56Z
+```
+
+Next, create the GatewayClass with the controller name "openshift.io/gateway-controller". This will instruct the Ingress Operator to install OSSM and configure it:
+```console
+$ oc create -f -<<'EOF'
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: GatewayClass
+metadata:
+  name: openshift-default
+spec:
+  controllerName: openshift.io/gateway-controller
+EOF
+gatewayclass.gateway.networking.k8s.io/openshift-default created
+```
+
+Behind the scenes, the Ingress Operator creates a ServiceMeshControlPlane (SMCP) CR, which is the configuration API for
+OSSM. Wait a minute or two for the SMCP to be created and become ready:
+```console
+$ oc get smcp -n openshift-ingress
+NAME                READY   STATUS            PROFILES      VERSION   AGE
+openshift-gateway   5/5     ComponentsReady   ["default"]   2.4.0     107s
+```
+
+OSSM will create two new Istio deployments in the `openshift-ingress` namespace. Ensure the pods for the deployments are running. These will need to become ready for the SMCP to also go ready:
+```console
+$ oc get deployment -n openshift-ingress
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+istio-ingressgateway       1/1     1            1           42s
+istiod-openshift-gateway   1/1     1            1           55s
+router-default             2/2     2            2           6h4m
+```
+
+Gateway API and OSSM have been installed successfully and are ready to use. We are now ready to create Gateways and HTTPRoutes.
+
+## Gateway API Example
+
+Let’s now use Gateway API to create a Gateway and Route. In this example, we will convert a simple HTTP OpenShift Route
+into roughly equivalent Gateway API objects (a Gateway and an HTTPRoute). This is the OpenShift Route that we will
+convert:
+```yaml
+apiVersion: v1
+kind: Route
+metadata:
+  name: http
+  namespace: demo-app
+spec:
+  host: demo.${DOMAIN}
+  to:
+    kind: Service
+    name: foo-app
+  port:
+    targetPort: 8080
+```
+
+First, create the Gateway with a Listener (`spec.listeners` instance) that will listen on a port for HTTP requests that
```

Comment:
This still has Listener capitalized. Maybe change the listener/Listener references throughout to be `listener` to signify the object reference?

Reply:
Good catch, I forgot this one. Now that I look again, even Gateway API [docs](https://gateway-api.sigs.k8s.io/concepts/api-overview/) aren't consistent (some Listeners some listeners). 

I'll update to lower case to be consistent.

Reply:
Good point, combined into one.